pkg/values/map.go

```diff
@@ -14,6 +15,40 @@ type Map struct {
 	Underlying map[string]Value
 }
 
+// TODO: this is temporary until the gateway can correctly unmarshal Web API trigger requests.
+func (m Map) MarshalJSON() ([]byte, error) {
```

Comment:
@kidambisrinivas This doesn't seem right to me -- what's making this necessary upstream?

Note you can marshal a map to json by unwrapping it, and then marshaling the resulting map[string]any. Can you do that wherever you need this instead?

Reply:
@jinhoonbang  confirmed that local testing works with those two methods commented out. seems like all gateway requests work. Removing this